exchange/exchangetest/request-multi-bidders-one-no-resp.json

```diff
@@ -0,0 +1,169 @@
+{
+  "incomingRequest": {
+    "ortbRequest": {
+      "id": "some-request-id",
+      "site": {
+        "page": "test.somepage.com"
+      },
+      "imp": [
+        {
+          "id": "my-imp-id",
+          "video": {
+            "mimes": ["video/mp4"]
+          },
+          "ext": {
+            "appnexus": {
+              "placementId": 1
+            },
+            "audienceNetwork": {
+              "placementId": "some-placement"
+            }
+          }
+        },
+        {
+          "id": "imp-id-2",
+          "video": {
+            "mimes": ["video/mp4"]
+          },
+          "ext": {
+            "appnexus": {
+              "placementId": 2
+            },
+            "audienceNetwork": {
+              "placementId": "some-other-placement"
+            }
+          }
+        }
+      ],
+      "ext": {
+        "prebid": {
+          "targeting": {}
+        }
+      }
+    }
+  },
+  "outgoingRequests": {
+    "appnexus": {
+      "mockResponse": {
+        "pbsSeatBid": {
+          "pbsBids": [
+            {
+              "ortbBid": {
+                "id": "winning-bid",
+                "impid": "my-imp-id",
+                "price": 0.71,
+                "w": 200,
+                "h": 250,
+                "crid": "creative-1"
+              },
+              "bidType": "video"
+            },
+            {
+              "ortbBid": {
+                "id": "losing-bid",
+                "impid": "my-imp-id",
+                "price": 0.21,
+                "w": 200,
+                "h": 250,
+                "crid": "creative-2"
+              },
+              "bidType": "video"
+            },
+            {
+              "ortbBid": {
+                "id": "other-bid",
+                "impid": "imp-id-2",
+                "price": 0.61,
+                "w": 300,
+                "h": 500,
+                "crid": "creative-3"
+              },
+              "bidType": "video"
+            }
+          ]
+        }
+      }
+    },
+    "audienceNetwork": {
+      "mockResponse": {
+        "pbsSeatBid": {
```

Comment:
I believe that if `pbsSeatBid` is initialized like this, it will be an array of zero length and so it won't reach the crash case without your fix. Instead, should this be ` "pbsSeatBid": null`?

Reply:
Fixed!